### PR TITLE
K3S Release September v2.7

### PR DESCRIPTION
**Pull request by matttrach**

### CODE REVIEW

**Review by rancher-max — COMMENTED** [review]
Are the minChannelServerVersion's here correct? I'd generally expect v1.22 to have `v2.6.3-alpha1`, v1.23 to have `v2.6.4-alpha1`, and v1.24 to have `v2.6.7-alpha1`.

It looks correct to me for v1.25 but I'm not 100% if it should be v2.7.0 or v2.7.1. It might make sense to wait on including that one at all until Rancher is more setup for 1.25.

**Comment by rancher-max** [comment]
https://github.com/rancher/rancher/issues/39083

**Comment by matttrach** [comment]
sorry, I was going off of the default branch name, I can change them.

**Review by rancher-max — CHANGES_REQUESTED** [review]
will you rebase this on the current dev-v2.7? It looks like it’s missing some things from there.

also… we'll need to update `maxChannelServerVersion` for all of these versions to be `v2.7.99`. And update the default version at the top of channels.yaml.

File: channels.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 channels:
   - name: default
-    latest: v1.22.11+k3s2
+    latest: v1.22.15+k3s2
```
Comment (rancher-max) [inline channels.yaml]: `+k3s1` here then should be good to go!

**Review by rancher-max — CHANGES_REQUESTED** [review]

File: channels.yaml (hunk 1 above)
Comment (matttrach) [reply channels.yaml]: done

**Review by rancher-max — APPROVED** [review]
Perfect! LGTM

**Comment by brandond** [comment]
Do we need to update the maxChannelServerVersion for all the old releases as well? Otherwise they won't be installable on 2.7. Or is that the desired behavior?

**Comment by rancher-max** [comment]
> Do we need to update the maxChannelServerVersion for all the old releases as well? Otherwise they won't be installable on 2.7. Or is that the desired behavior?

It was confirmed the other day that just the one previous version is okay. Ref: https://github.com/rancher/kontainer-driver-metadata/pull/978#issuecomment-1258710173

**Review by brandond — APPROVED** [review]

**Pull request merged**